### PR TITLE
Remove unused reverse proxy

### PR DESCRIPTION
**Pull request by matthias-ronge**
The reverse proxy is a relic from Production 2.x. It is no longer needed and can be removed.

### CODE REVIEW

File: Kitodo/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -88,15 +88,6 @@
...
-    <filter>
-        <filter-name>Proxy</filter-name>
-        <filter-class>net.sf.j2ep.ProxyFilter</filter-class>
-        <init-param>
-            <param-name>dataUrl</param-name>
-            <param-value>/WEB-INF/reverse-proxy.xml</param-value>
-        </init-param>
-    </filter>
-
     <filter-mapping>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/WEB-INF/web.xml]: Is this entry still needed after you removed the proxy?

Comment (matthias-ronge) [reply Kitodo/src/main/webapp/WEB-INF/web.xml]: You're right. Thanks for the hint

**Comment by Kathrin-Huber** [comment]
please resolve conflicts

**Comment by Kathrin-Huber** [comment]
I think this is already removed by #3071

**Comment by henning-gerhardt** [comment]
Is this pull request still needed? This changes should be superseeded by #3071 .

**Comment by matthias-ronge** [comment]
Then that's probably double. Well, two heads, one thought.

**Pull request closed without merging**